src/bundle/settings.rs

```diff
@@ -44,15 +44,22 @@ struct PackageSettings {
     metadata: Option<MetadataSettings>,
 }
 
+#[derive(Clone, Debug, Deserialize)]
+struct WorkspaceSettings {
+    members: Option<Vec<String>>
+}
+
 #[derive(Clone, Debug, Deserialize)]
 struct CargoSettings {
-    package: PackageSettings,
+    package: Option<PackageSettings>, // "Ancestor" workspace Cargo.toml files may not have package info
+    workspace: Option<WorkspaceSettings>, // "Ancestor" workspace Cargo.toml files may declare workspaces
 }
 
 #[derive(Clone, Debug)]
 pub struct Settings {
-    cargo_settings: CargoSettings,
-    package_type: Option<PackageType>, // If `None`, use the default package type for this os
+    package: PackageSettings,
+    package_type: Option<PackageType>,
+    // If `None`, use the default package type for this os
```

Comment:
It's a little confusing to have this comment here, since it looks like it applies to the `target` field.  If it won't fit on the end of the `package_type` line, then could we move it to just above that line?

Reply:
Hmm... seems like it's the IntelliJ automatic formatting that pushed it to a new line
(a new line that doesn't make sense IMO).... so will put it back manually.